src/GNDStk/Tree/test/decl.test.cpp

```diff
@@ -0,0 +1,107 @@
+
+#include "catch.hpp"
+#include "GNDStk.hpp"
+
+SCENARIO("Testing GNDStk tree decl()") {
+   using namespace njoy::GNDStk;
+   using namespace child;
+
+   GIVEN("A tree read from n-069_Tm_170-covar.xml") {
+      Tree<> t("n-069_Tm_170-covar.xml");
+
+      /*
```

Comment:
Maybe it would become incorrect. But maybe not, considering that we did choose to include a handful of GNDS files in the repository, and I'm not sure there's any reason those would change, over time, in terms of the test. I'd put this in there (note also that it's abbreviated - not the full XML file contents) so someone could have this one file loaded up in one window on the screen in front of them, and easily see how the various queries align with the XML contents. I suppose that, yes, that could also be done by putting the XML file contents into a literal string in the .cpp file. In general, though, I don't want to do that, because we have those GNDS files in the repo already, and they're used in lots of test files - we wouldn't want to replicate them across so many test files.